### ISSUE TITLE
Doesn't find async actions

### ISSUE THREAD

**Issue by knownasilya** [DockYard/ember-route-action-helper#65]
For some reason, this helper doesn't find async actions, e.g.

```hbs
actions: {
  async submit() {
    await doSomething();
    await doAnotherThing();
  }
}
```

The above works if using just `(action 'submit'` in another context.

**Comment by poteto** [comment]
What version are you using? This should have been fixed in #60

**Comment by knownasilya** [comment]
v2.0.2. Looks like that merge is still in master, not released yet.

**Comment by poteto** [comment]
Released `2.0.5`

**Comment by knownasilya** [comment]
Sweet, thanks a bunch!
